service_identity/src/main/kotlin/org/thechance/service_identity/domain/usecases/user_address/IManageUserAddressUseCase.kt

```diff
@@ -0,0 +1,44 @@
+package org.thechance.service_identity.domain.usecases.user_address
```

Comment:
you can put the kt file on usecases directly

Reply:
Ok, i will refactor it